dot/services.go

```diff
@@ -182,9 +182,13 @@ func asAuthority(authority bool) string {
 	}
 	return ""
 }
-
-func createBABEService(cfg *Config, st *state.Service, ks keystore.Keystore,
-	cs *core.Service, telemetryMailer telemetry.Client) (*babe.Service, error) {
+func (nb nodeBuilder) createBABEService(cfg *Config, st *state.Service, ks keystore.Keystore,
+	cs *core.Service, telemetryMailer telemetry.Client) (babe.ServiceIFace, error) {
+	return nb.createBABEServiceBuilder(cfg, st, ks, cs, telemetryMailer, babe.Builder{})
```

Comment:
There is some naming issue here; `createBABEServiceBuilder` should be `createBABEServiceWithBuilder` I think.

Reply:
Renamed.